lib/object.js

```diff
@@ -656,6 +656,8 @@ internals.Object = class extends Any {
             }
 
             return this.createError('object.type', { type: name }, state, options);
+        }, {
```

Comment:
This is not the place to put it, the 2nd argument of _test holds the describe data.